internal/util/debug/debug.go

```diff
@@ -44,7 +44,7 @@ const (
 )
 
 // RunHandler runs debug handler.
-func RunHandler(ctx context.Context, addr string, r prometheus.Registerer, l *zap.Logger) {
+func RunHandler(ctx context.Context, addr string, r prometheus.Registerer, l *zap.Logger, started <-chan struct{}) {
```

Comment:
It's mainly because of the usage of the `RunHandler` function inside of `envtool` and `integration` package, where we don't check the kubernetes probes at all.

Reply:
Can we pass a closed channel for  `envtool` and `integration` packages? Then it returns `http.StatusOK` without having special handling for nil.

```
started := make(chan struct{})
close(started)
go debug.RunHandler(ctx, "127.0.0.1:8089", prometheus.DefaultRegisterer, logger.Named("debug").Desugar(), started)
```

Reply:
sure